x/savings/keeper/grpcquery_test.go

```diff
@@ -0,0 +1,193 @@
+package keeper_test
+
+import (
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	tmprototypes "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/savings/keeper"
+	"github.com/kava-labs/kava/x/savings/types"
+)
+
+type grpcQueryTestSuite struct {
```

Comment:
test suite doesn't seem to be actually run, missing 

```go
func TestGrpcQueryTestSuite(t *testing.T) {
	suite.Run(t, new(grpcQueryTestSuite))
}
```

Reply:
Good catch

Reply:
Updated in Updated in https://github.com/Kava-Labs/kava/pull/1193/commits/a944cc35ceeb67a5c57c41b3d59bc1231bd5c6c8